src/Tests/UnitTest/ComInteropTests.cs

```diff
@@ -161,5 +162,12 @@ public void TestDisplayInformation()
             Assert.Throws<COMException>(() => DisplayInformationInterop.GetForWindow(new IntPtr(0)));
             Assert.Throws<COMException>(() => DisplayInformationInterop.GetForMonitor(new IntPtr(0)));
         }
+
+        [Fact]
+        public void TestDataTransferManager()
+        {
+            Assert.Throws<COMException>(() => DataTransferManagerInterop.GetForWindow(new IntPtr(0)));
+            DataTransferManagerInterop.ShowShareUIForWindow(new IntPtr(0));

```

Comment:
We should Assert.Throws<COMException> here.  Latest CI run showed the below exception here which is expected and also shows the function is being called:

System.Runtime.InteropServices.COMException : Invalid window handle. (0x80070578)

Reply:
> 我们应该在这里 Assert.Throws 类似于上一行。最新的 CI 运行在此处显示以下异常，这是预期的，并且还显示正在调用函数：
> 
> System.Runtime.InteropServices.COMException：无效的窗口句柄。（0x80070578）

![image](https://github.com/microsoft/CsWinRT/assets/49179966/3bb8df9c-ad60-4b4e-a598-dcdca5a4698b)
I tried to write this test method with reference to IPlayToManagerInterop because the two interfaces have similar function call names and parameters. As you said just now, if in DataTransferManagerInterop.ShowShareUIForWindow add test statements, does that also should add a new PR, In PlyaToManagerInterop.ShowPlayToUIForWindow() to add similar testing statement?

Reply:
> We should Assert.Throws here similar to the previous line. Latest CI run showed the below exception here which is expected and also shows the function is being called:
> 
> System.Runtime.InteropServices.COMException : Invalid window handle. (0x80070578)

![image](https://github.com/microsoft/CsWinRT/assets/49179966/3bb8df9c-ad60-4b4e-a598-dcdca5a4698b)
I tried to write this test method with reference to IPlayToManagerInterop because the two interfaces have similar function call names and parameters. As you said just now, if in DataTransferManagerInterop.ShowShareUIForWindow add test statements, does that also should add a new PR, In PlyaToManagerInterop.ShowPlayToUIForWindow() to add similar testing statement?

Reply:
That other test isn't failing, so I would leave it as is.  It is possible that API handles invalid handle without erroring whereas this one doesn't.

Reply:
> That other test isn't failing, so I would leave it as is. It is possible that API handles invalid handle without erroring whereas this one doesn't.

Well. Thank you for your answer.